apps/frontend/package.json

```diff
@@ -66,7 +66,7 @@
   "proxy": "http://localhost:4000",
   "scripts": {
     "dev": "vite",
-    "dev:docker": "npm install && vite",
+    "dev:docker": "npm install && vite --no-open",
```

Comment:
This potentially override config in the `vite.config `can we do it there if possible. It effectively closes https://github.com/UserOfficeProject/user-office-project-issue-tracker/issues/975 also.

Reply:
Yeah it overrides `vite.config`, I assumed that ESS wanted that to stay as is so I only overrode it when launching in our dev enviornment. Happy to change it over there instead

Reply:
Okay happy for it to go in as is.